rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/NodeSourceWindow.java

```diff
@@ -106,8 +105,14 @@
 
     private static final String HIDDEN_INFRA = "hidden-infra";
 
+    // mark whether the form item correspond an important field (i.e., whether it should be shown when hiding advanced configuration)
```

Comment:
Maybe `required field` (vs optional) would be more relevant than important

Reply:
an important field is not necessarily required. It simply means that it's a field cpmmonly modified, but not all important fields need to have a value